go.mod

```diff
@@ -4,6 +4,7 @@ go 1.20
 
 require (
 	github.com/Mellanox/rdmamap v1.1.0
+	github.com/container-orchestrated-devices/container-device-interface v0.5.4
```

Comment:
reminder to check if we prefer to:

1. bump this to v0.6.0 and require containerd 1.7.5 or cri-o 1.28 or keep as is.

Reply:
According to https://github.com/containerd/containerd/blob/v1.7.5/go.mod#L10, we need to keep v0.5.4 now